src/client/a2/main/grid/layers/filters/radar-filter.ts

```diff
@@ -0,0 +1,88 @@
+import * as PIXI from 'pixi.js'
+
+export class RadarFilter extends PIXI.Filter {
+  constructor(private maxTime: number) {
+    super(
```

Comment:
Looks like these parameters are meant to be used to load an external file, not inline a bunch of code